### ISSUE TITLE
Global CSS import does not handle backslashes correctly

### ISSUE THREAD

**Issue by jsardev** [yuanqing/create-figma-plugin#209]
I wanted to use Tailwind CSS [arbitrary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) and it did not work correctly with current `create-figma-plugin` implementation.

The problem is that Tailwind outputs properly escaped `[`/`]` signs in css, but it is not handled correctly in the inline import logic.

Consider this `output.css` file
```css
.grid-cols-\[1fr_1fr\] {
  grid-template-columns: 1fr 1fr;
}
```

After importing it via `import '!./output.css'` we get:
```css
.grid-cols-[1fr_1fr] {
  grid-template-columns: 1fr 1fr;
}
```

Which is incorrect CSS - the `[`/`]` values are considered atrribute selectors and not strings.

### Additional info

Version: `3.1.0`

**Comment by arkus-pm** [comment]
As for some reason [210](https://github.com/yuanqing/create-figma-plugin/pull/210) is not merged for a month and [182](https://github.com/yuanqing/create-figma-plugin/pull/182) is not merged for 6 months, here's temporary fix for arbitary values in Tailwind:

❗️ replace `tailwind` in `filter: /tailwind\.js$/` with the name of your tailwind output file, which you import in your tsx files.

```
/* build-figma-plugin.ui.js */

const { readFileSync } = require("fs")

function fixTailwindInjectionPlugin() {
  return {
    name: "fix-tailwind-injection-plugin",
    setup(build) {
      build.onLoad({ filter: /tailwind\.js$/ }, async (args) => {
        const contents = readFileSync(args.path, "utf8")

        const modifiedContents = contents
          .replaceAll("\\2c ", "\\\\,")
          .replaceAll("\\[", "\\\\[")
          .replaceAll("\\]", "\\\\]")

        return { contents: modifiedContents, loader: "js" }
      })
    },
  }
}

module.exports = function (buildOptions) {
  return {
    ...buildOptions,
    define: {
      global: "window",
    },
    plugins: [
      ...buildOptions.plugins
      fixTailwindInjectionPlugin(),
    ],
  }
}

```

**Comment by jsardev** [comment]
@arkus-pm Thank you for the workaround! ❤️

**Comment by pawelkontek** [comment]
@yuanqing could you merge this / fix this, pretty please? 🙏
It is bugging me, and I am not that versed in applying this change in my local setup.

**Comment by yuanqing** [comment]
@jsardev @arkus-pm @pawelkontek

I’ve just published a proper fix for this issue – **can I get your help to verify that it works on your local set up?** All you need to do is update all your `@create-figma-plugin/*` packages to version `"3.2.1-alpha.2"` in your `package.json`, re-install, run the `build` script, and see if things works as expected in your plugin

I plan to publish the fix in an official release (`3.2.1`) once I get confirmation that this bug has been fixed. Thanks!

**Comment by te6-in** [comment]
@yuanqing Just tested and it worked perfectly. Now I removed all the custom utility classes I used as a workaround. Thanks a lot for the fix!

**Comment by yuanqing** [comment]
This should be fixed as of `3.2.1`
